LoopKit/TherapySettings.swift

```diff
@@ -34,6 +34,20 @@ public struct TherapySettings: Equatable, Codable {
         return glucoseTargetRangeSchedule?.unit
     }
     
+    public var isComplete: Bool {
+        return
+            glucoseTargetRangeSchedule != nil &&
+            //preMealTargetRange != nil &&
+            //workoutTargetRange != nil &&
```

Comment:
[nit] remove?

Reply:
I kept those there as comments that they're purposefully not included in the check.

Reply:
+1 to Darin's comment.